apps/andi/lib/andi/schemas/dataset_upload.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Andi.Schemas.DatasetUpload do
+  @moduledoc """
+  Module representing a dataset upload request in postgres
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  alias Andi.InputSchemas.StructTools
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+  schema "dataset_upload" do
+    field(:dataset_id, Ecto.UUID)
+    field(:timestamp, :utc_datetime)
+    field(:user_uploading, :string)
+    field(:upload_success, :boolean)
```

Comment:
Anything else anyone would want to save here?

Reply:
File name?

Reply:
MD5 checksum?